lib/susedistribution.pm

```diff
@@ -665,7 +665,8 @@ sub hyperv_console_switch {
     # If we are in VT, 'Alt-Fx' switch already worked
     return if check_screen('any-console', 10);
     # We are in X11 and wan't to switch to VT
-    testapi::x11_start_program('xterm');
+    wait_still_screen { type_string "xinit /usr/bin/xterm -- :1 &\n" };
+    testapi::mouse_set(10, 10);
     self->distribution::script_sudo("exec chvt $nr; exit", 0);
```

Comment:
All following commands may fail if "xinit /usr/bin/xterm -- :1 &\n" fails, but execution will not stop at the right place because there is no proper assertion.

Reply:
Will use the method assert_still_screen instead of wait_still_screen.